src/mpid/ch4/netmod/ofi/subconfigure.m4

```diff
@@ -255,6 +259,10 @@ AM_COND_IF([BUILD_CH4_NETMOD_OFI],[
             "netdir")
                 AC_DEFINE([MPIDI_CH4_OFI_USE_SET_RUNTIME], [1], [Define to use runtime capability set])
                 enable_netdir="yes"
+            "verbs;ofi_rxm")
```

Comment:
I think the semicolon is causing syntax error.

Reply:
Scratch that, it is missing `;;` in the above case.

Reply:
The FI_PROVIDER string needs to be in double-quotes otherwise it throws syntax error.
FI_PROVIDER="verbs;ofi_rxm"

Reply:
Ignore, above comment. Did not see your second comment. Will fix the case.